package.json

```diff
@@ -51,9 +50,7 @@
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.24.1",
     "bundlesize": "^0.5.7",
-    "commitizen": "^2",
```

Comment:
Yeahhhh, i feel like 90% of the time when taking contributions this is done manually in the squash & merge message. Fair removal 👍